src/components/AssetSearch/AssetSearch.test.tsx

```diff
@@ -64,6 +66,40 @@ describe('AssetSearch', () => {
       done();
     });
   });
+  it('should call onSearchResult when it is defined', done => {
+    const { onSearchResult } = propsCallbacks;
+    const showLiveSearchResults = false;
+    const props = { onSearchResult, showLiveSearchResults };
+    const wrapper = mount(<AssetSearch {...props} />);
+
+    wrapper
+      .find(Input)
+      .find('input')
+      .simulate('change', { target: { value: 'value' } });
+
+    setImmediate(() => {
+      wrapper.update();
+      expect(onSearchResult).toHaveBeenCalled();
+      done();
+    });
+  });
+  it('should call onSearchResult with empty array in parameter when input is an empty string', done => {
+    const { onSearchResult } = propsCallbacks;
+    const showLiveSearchResults = false;
+    const props = { onSearchResult, showLiveSearchResults };
+    const wrapper = mount(<AssetSearch {...props} />);
+
+    wrapper
+      .find(Input)
+      .find('input')
+      .simulate('change', { target: { value: '' } });
+
+    setImmediate(() => {
+      wrapper.update();
+      expect(Array.isArray(onSearchResult.mock.calls[0][0])).toBeTruthy();
```

Comment:
Could you change this test? You're expecting an empty array as a result in case of empty string input (not for an array type itself)

Reply:
sure, fixed